docs/design/Abstract Syntax Tree.md

```diff
@@ -213,926 +169,13 @@ following:
 The `_: MetaContainer` component here simply causes a compile-time assertion that the third property of
 `VariableReference` is of type `MetaContainer`, resulting in a compile-time error whenever a new property is added.
 
-### Arbitrary Meta Information Can Be Attached to Any Node
-
-TODO:  leaving this part unspecified for the moment because there is still some uncertainty surrounding the how metas
-work.
+3. Arbitrary Meta Information Can Be Attached to Any Node
 
+TODO:  leaving this part unspecified for the moment because there is still some uncertainty surrounding how metas work.
 
-## Rules for working with the AST
+4. Rules for working with the AST
 
 - Always add new properties *before* the `metas: MetaContainer` of a new property.
 - When using `when` with a sealed class and branching by derived types as a statement use `case()` and `toUnit()` to
   trick the Kotlin compiler into being helpful.
 - When using `when` with the derived types of a sealed class, use destructuring in each branch where possible.
-
```

Comment:
The removed part from this line below is for V0 & V1 AST.

Reply:
What do you want that doc for? Is it for migration from V0 to PIG AST?

Reply:
By the way, V1 AST is not used by anyone from what I know

Reply:
More like examples of PIG generated AST.

Reply:
Here, https://github.com/partiql/partiql-lang-kotlin/issues/898